pkg/config/config.go

```diff
@@ -10,11 +11,20 @@ import (
 func NewFromInputs(a *githubactions.Action) (*Config, error) {
 	cfg := Config{}
 
-	cfg.Command = a.GetInput(commandInput)
-	cfg.AccessToken = a.GetInput(accessTokenInput)
-	cfg.ProjectId = a.GetInput(projectIdInput)
-	cfg.ScriptDir = a.GetInput(scriptDirInput)
-	cfg.Function = a.GetInput(functionInput)
+    command := os.Getenv("GAS_COMMAND")
+    if ( command == "" ) {
```

Comment:
```suggestion
    if command == "" {
```

brackets are not needed ;)

Reply:
I removed them. Go novice :)